lib/main_common.pm

```diff
@@ -723,9 +723,13 @@ sub load_common_installation_steps_tests {
 }
 
 sub load_inst_tests {
+    # On SLE 15 dud addon screen is shown before product selection
+    if (get_var('DUD_ADDONS') && is_sle('15+')) {
```

Comment:
I am sure this is not sle - specific. Better turn around and only exclude old sle-versions

Reply:
DUD_ADDONS is sle specific and is used only in dud_sdk/dev_tools scenario. I guess you are referring to DUD variable. So, in the end it would be more relevant for the change below if at all.